homeassistant/components/notify/html5.py

```diff
@@ -144,6 +157,21 @@ def _load_config(filename):
     return {}
 
 
+class HTML5ApplicationServerKeyView(HomeAssistantView):
```

Comment:
We should make this a websocket command. We are trying to phase out the usage of HTTP endpoints.

Reply:
Here is an example command https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/config/auth.py#L27-L29